text/2019-11-01-tikv-community-reorg-sig.md

```diff
@@ -0,0 +1,148 @@
+
+In addition, this picture also reflects the relationship between community roles and SIGs.
+
+- Active Contributor
+    + Must be a Contributor with at least 8 PRs merged to the master branch within 1 year.
+    + To join an SIG, a Contributor needs to contribute more than 8 PRs to the module that the SIG is responsible for within 1 year, so that he/she can be invited to join the SIG for targeted learning and contributions.
+
+- Reviewer
+    + Must belong to a SIG; has the obligation to review PRs.
+  + Generated from Active Contributors. When an Active Contributor has significant contributions to the module and is nominated by 2 or more Committers, he/she will be invited to be a Reviewer of the module.
+    + Has valid sign-off (approval with Looks Good To Me, or LGTM) for code reviews of the module
+
+- Tech Lead
+    + The organizer of the SIG; responsible for the day-to-day operations including organizing meetings, answering questions, etc.
+    + Tech Leads are responsible for the management and growth of the SIG. For now, the role is taken by PingCAP members, but is hopefully extended to community developers in the future.
+
+The constitution and operation of SIGs will be detailed in later sections.
+
+#### Working Group
+
+The third change is the introduction of the Working Group (WG) in the developer community structure. A WG is formed by community developers who gather together to accomplish a specific goal with PingCAP members. To achieve the goal, some WGs may span over multiple SIGs, and some WGs may only focus on something specific in a specific SIG.
+
+Each WG has its life cycle. Once the goal is completed, the group will be disbanded. The only goal of WG operations and management is to ensure that the goals set by the group are completed at the right time. In general, the working group hold periodic meetings to summarize the current project progress and determine the plans for next steps.
+
+### SIG Constitution and Operation
+
+SIGs mainly aggregate a group of Active Contributors, and conducts in-depth research & contributions to one or more TiKV modules, and is promoted to Reviewer, Committer within the SIG.
+
+#### SIG role and organizational governance
+
+The SIG follows the role definitions in the Community structure. The SIG is subject to the following rules:
+
+- Before starting a SIG, the constitution will be established in advance. The detailed steps of creating the constitution can be found on the corresponding document.
+- Except for holidays, the regular meeting is organized at least once every two weeks.
+- The assignable tasks will be graded by difficulty levels and claimed by SIG members.
+
+#### SIG Roles
+
+- **Active Contributors**
+
+    - Active contributors will be invited to the SIG
+    - Keep active inof at least one module in the SIG
```

Comment:
Sorry for my ignorant, that does inof mean exactly?

Reply:
Sorry, a typo. Now fixed.